presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestInsertIntoHiveTable.java

```diff
@@ -122,4 +124,34 @@ public void testInsertIntoSelectToHiveTableAllHiveSimpleTypes()
                         true,
                         "kot binarny".getBytes()));
     }
+
+    @Test(groups = {HIVE_CONNECTOR, POST_HIVE_1_0_1})
+    public void testInsertIntoPartitionedWithSerdePropety()
+    {
+        String tableName = "target_partitioned_with_serde_property";
+        query(format("DROP TABLE IF EXISTS %s", tableName));
+        String query = format(
+                "CREATE TABLE %s(" +
```

Comment:
Convert this into mutable table `Requirement`, see `com.facebook.presto.tests.convention.TestShowPartitions#getRequirements` for example.